src/helpers/StripUnit.js

```diff
@@ -1,14 +1,26 @@
+/** @module Helpers */
+import isInteger from '../internalHelpers/_isInteger'
+
 /**
- * Strips the unit from a given value.
- * @param {string} value - a string that is a number plus a unit of measure.
- * @return {number} - an integer without the unit.
+ * CSS to represent truncated text with an ellipsis.
+ * @static
+ * @param {string} [width = 100%] - Max-width for text to respect before being truncated.
+ * @return {Object} CSS as JS for truncating text.
```

Comment:
Good catch. This is just an artifact of me testing the doc generator across multiple items.  I'll add proper documentation for this.